app-backend/db/src/test/scala/com/azavea/rf/database/PlatformDaoSpec.scala

```diff
@@ -37,16 +39,20 @@ class PlatformDaoSpec extends FunSuite with Matchers with IOChecker with DBTestC
         result.name shouldBe "test platform"
     }
 
-    test("insertion types") { check(PlatformDao.selectF.query[Platform]) }
+    test("insertion types") {
+        createPlatform.map(p => check(PlatformDao.createF(p).query[Platform]))
```

Comment:
this is... a `ConnectionIO[TestResult]`?

Reply:
It turns out the `check` on the inside is being run as part of the tests, even though the test itself is a `ConnectionIO` something-or-other. We proved this by ruining the query (flipping columns, referring to a bogus table) and observing failures.

Reply:
Wait actually we proved the opposite and I misread the test results -- oops! anyway this isn't actually being run